react/src/lib/DatePicker/DatePickerCalendar/index.js

```diff
@@ -87,7 +87,7 @@ class DatePickerCalendar extends React.Component {
       const allPrevDaysDisabled = shouldPrevMonthDisable(date, minDate);
       return (
         <Icon
-          ariaLabel={previousArialLabel}
+          ariaLabel={`${previousArialLabel} month, ${subtractMonths(date.clone(), 1).format("MMMM")}`}
```

Comment:
My concern here is internationalization.  If we don't give them control to replace the string completely than month will remain month always.  Perhaps we could have an internal variable that sets this value if previousAriaLabel prop isn't present?